docs/server.rst

```diff
@@ -139,8 +139,16 @@ If we assume that you are installing Tunneldigger under ``/srv/tunneldigger``
     cd /srv/tunneldigger
     virtualenv env_tunneldigger
```

Comment:
Might make sense to add a `-p` here as well?

Reply:
Reworded this whole thing in the new push.